### DISCUSSION TITLE
Digital CPU & GPU Temp and Load (Landscape) [3.5"]

### DISCUSSION THREAD

**Discussion by nexpid** [mathoudebine/turing-smart-screen-python#538] [Themes]
A modified verison of @geekm0nkey's [BIG Digital Temps (portrait) [3.5"]](https://github.com/mathoudebine/turing-smart-screen-python/discussions/296) in landscape version and an extra clock
The clock can be disabled by changing `True` to `False` at line 76 in `theme.yaml`

```diff
  75   # Set below to False to remove the clock
- 76   SHOW: True
+ 76   SHOW: False
```
![preview](https://github.com/user-attachments/assets/5e875f3b-0017-49cb-b798-880659246322)
![screencap](https://github.com/user-attachments/assets/2dc06922-17e0-4f3c-b382-8c7c280f6846)

**Comment by oopiicaa** [comment]
Hmm, there seems to be something wrong with theme.yaml: in the CPU TEMP column the value is not shown - instead the yellow digits of cpu temperatures are shown in GPU TEMP column. GPU TEMP values are not shown.

![1721630321862](https://github.com/user-attachments/assets/a1d5d5c6-656a-47ea-92e9-96a6d2cc4460)

**Reply by nexpid** [reply]
Have you tried updating the repo? Maybe your local copy is outdated. Try running
```sh
git pull
```

**Reply by oopiicaa** [reply]
You mean turing-smart-screen-python repo? I have the latest version. I actually had to change X-coordinate values for CPU and GPU temperatures.
In yaml from https://github.com/user-attachments/files/16329444/DigitalLandscape_3.zip the X value for CPU was `240` and for GPU `469`. I had to change them to `10` and `240` to get it to show properly for me.

![image](https://github.com/user-attachments/assets/0695333a-053d-4b69-a0f2-5a0c953275da)

**Comment by nexpid** [comment]
Updated the theme to version 3! Fixed an issue with the clock misaligning on some devices when hour wasn't double digits. Make sure to update

**Comment by DJRoby19** [comment]
Hi, I installed your theme but I can't see the GPU temperature or the load on version 3.5.0, any ideas?
I don't understand why some themes work the gpu and on others not for example on the Cyberdeck theme everything works except the gpu fan speed

**Reply by DJRoby19** [reply]
So I found you error because it does not display the Gpu must be like this GPU:
INTERVAL: 1
PERCENTAGE:

**Reply by LakesideMiners** [reply]
can you post the config.yml please?

**Reply by DJRoby19** [reply]
There he is

[config.yaml.txt](https://github.com/user-attachments/files/17076266/config.yaml.txt)

**Reply by LakesideMiners** [reply]
gah sorry, i ment the theme one that you fixed

**Comment by DJRoby19** [comment]
Ok here it is
[DigitalLandscape.zip](https://github.com/user-attachments/files/17084077/DigitalLandscape.zip)

**Comment by Bluslaw** [comment]
How to add this? Software require a .data files

**Reply by Akinoph** [reply]
I ask same question

**Reply by c0ldJS** [reply]
@Bluslaw & @Akinoph: You're using the stock software for your displays. This repo/project is a replacement and an alternative for that software.

You'll need to use your software's theme editor option to replicate the look and feel.
